libs/librepcb/projecteditor/boardeditor/fsm/bes_addvia.cpp

```diff
@@ -283,45 +292,65 @@ BES_Base::ProcRetVal BES_AddVia::processSceneEvent(BEE_Base* event) noexcept {
 }
 
 bool BES_AddVia::addVia(Board& board) noexcept {
-  Q_ASSERT(mUndoCmdActive == false);
-  Q_ASSERT(mCurrentViaNetSignal);
+  Q_ASSERT(mSubState == SubState_Idle);
+  Q_ASSERT(!mUndoStack.isCommandGroupActive());
```

Comment:
This assertion fails (and thus LibrePCB crashes) if another tool (for example in the schematic editor) is active when the "add via" tool is started. It's totally legal to try starting a tool while another command is active, thus this assertion should be removed:

```suggestion
```